lib/pltr/v2/migrator/migrations/__tests__/2021.6.9.js

```diff
@@ -0,0 +1,31 @@
+import migration from '../2021.6.9'
+
+jest.mock('format-message')
+
+describe('2021.6.9', () => {
+  const { state_2021_6_9 } = require('./fixtures')
+  const initialFile = state_2021_6_9
+  describe('given the initial file state', () => {
+    it('should produce the initial file state', () => {
+      expect(migration(initialFile)).toEqual(initialFile)
+    })
+  })
+  describe('given a state with hierarchyLevel in it', () => {
+    const { hierarchyLevel } = require('../../../store/initialState')
+    const stateWithHierarchy = {
+      ...initialFile,
+      hierarchyLevels: { hierarchyLevel },
```

Comment:
This schema isn't correct.  It should be: `{ 0: hierarchyLevel }`.
Curiously, the test will still work because the migration doesn't care what the keys are called.
It's important to correct this because people use tests as documentation for how things are supposed to work.